### ISSUE TITLE
Generation of DL Model 💻 & Label Encoder ⚙️

### ISSUE THREAD

**Issue by ackwolver335** [ackwolver335/MusicGenreClassification#2]
# 🖥️ Deep Learning Model Generation with Encoder

- 📦 Packages used → **Tensorflow**, **Keras**, **Seaborn**, **Matplotlib**, **Numpy** & **Pandas**.
- Generation of Label Encoder using **Pickle** Module.
- 📩 Generation Output → **Model's Summary**, **Classification Report** and **Confusion Matrix**.
- Encoder contains specific different files.

**Comment by ackwolver335** [comment]
## Model & Encoder Generated Successfully

- DL Model Generated with its extension **.keras**
- Label Encoder for finding specific traits of Music Genre.
